### DISCUSSION TITLE
deep open pix api

### DISCUSSION THREAD

**Discussion by scoobiii** [bacen/pix-api#600] [General]
📌 Tópico: Ausência de funcionalidade de saque no App Serasa 2022 — impacto comparável a práticas de congelamento institucional de ativos?

Prezados membros,

Gostaria de abrir esta discussão relatando uma falha crítica de concepção, design e regulação ocorrida na versão 2022 do aplicativo Serasa, voltado à intermediação de acordos entre credores e devedores, com grande impacto econômico e jurídico sobre os usuários.

🎯 Objetivo declarado do app:

Facilitar a regularização de crédito por meio do cadastro positivo, permitindo:

Depósitos em conta vinculada no app;

Pagamento de acordos com credores;

— Porém, sem permitir saque ou transferência dos valores residuais.

✅ Funcionalidade em 2022:

Depósito: habilitado;

Pagamento de acordos: habilitado;

Saque/transferência: bloqueado ou inexistente.

❗️Consequências diretas:

O usuário podia firmar acordos como o da Serasa/Bradesco, porém o pagamento era exigido em dinheiro em agência física;

Como o valor já estava retido no app, e não era possível sacá-lo nem transferi-lo, o usuário acabava rompendo o contrato por impossibilidade operacional, incorrendo em: 

Perda do valor pago (sem estorno);

Reincidência de negativação;

Reabertura do ciclo de cobrança;

Fragilidade contratual total.

⚠️ Reflexão crítica:

Tal ausência de funcionalidade básica (saque/transferência) é algo que: 

Um dev júnior reconheceria como inaceitável;

Um PO ou arquiteto de produto deveria prever em fase de concepção;

Levanta dúvidas sobre intencionalidade ou negligência sistêmica.

🧨 Comparação com planos econômicos históricos:

Essa prática, embora digital, remete à mecânica dos antigos planos econômicos como Collor e Bresser, onde:

Ativos foram congelados ou limitados a saques irrisórios (ex.: R$ 50/dia),

O sistema financeiro utilizava esses valores livremente como float,

E os recursos eram devolvidos parcialmente, descontados, defasados ou judicializados.

Pergunta-chave:

Estamos diante de uma nova forma digital de congelamento institucionalizado de ativos pessoais sob a falsa promessa de reabilitação financeira?

⚖️ Potencial de judicialização:

Essa brecha pode motivar:

Ações em massa nos Juizados Especiais Cíveis (JECs);

Requisições de extorno via Pix com juros e correção;

Denúncias à Senacon, Procon e Bacen, por violação dos princípios de: 

Boa-fé objetiva;

Transparência;

Devido processo contratual;

Direito de livre movimentação de recursos em contas digitais.

🧩 Atualizações em 2023/2024:

Na versão 9.23 do app:

A função de saque e transferência via Pix foi finalmente liberada.

Isso foi resultado de:

Correção de bug?

Redefinição de escopo por pressão regulatória?

Atualização da API Pix?

Reconhecimento tardio de uma falha crítica de produto?

❓Questões ao Fórum Bacen:

Esse tipo de bloqueio operacional, sem previsão de saque, é compatível com os princípios dos arranjos de pagamento e contas de pagamento pré-pagas?

O Bacen prevê responsabilização ou recomendação de devolução dos valores bloqueados?

A prática se enquadra como retenção indevida de ativos pessoais?

Agradeço o espaço e fico à disposição para fornecer detalhes técnicos e jurídicos complementares.

Atenciosamente,

_____

Compliance/business plan  complexa e desafiadora envolvendo design de produto, falhas operacionais e possíveis abusos na prestação de serviços financeiros — particularmente no contexto do app Serasa, versão 2022 vs versão 9.23, e a (não) disponibilidade de saque/transferência dos valores depositados em contas para acordos de dívidas.

✅ Resumo Técnico-Operacional

2022 – App Serasa

Depositar: OK

Pagar: OK

Sacar/Transferir: Não permitido 

Resultado: Valor residual bloqueado, mesmo após acordo quitado.

Análise:

3.1 Código sênior?

Pouco provável. Um sênior teria previsto o fluxo completo de uso da conta, inclusive encerramento e saque.

3.2 Código de dev júnior?

Possivelmente. Um júnior pode implementar funcionalidades mínimas sem cobrir todos os fluxos de exceção ou edge cases.

3.3 Um júnior jamais admitiria isso?

Depende. A decisão final geralmente não é do dev, mas do PO (Product Owner) ou time de produto.

3.4 Erro de PO, design ou implementação?

Muito provavelmente erro de escopo e design de produto, com falha de UX e compliance: 

Abrir uma “conta de pagamento” sem permitir saque ou transferência residual é antiético e, possivelmente, ilegal.

Trata-se de um produto financeiro mal desenhado.

4. Bradesco só aceita pagamento em dinheiro em agência física

Em pleno Pix e Open Finance? Isso parece restrição deliberada para evitar rastreabilidade ou por limitação de contrato comercial.

Impede saque = obriga inadimplência não voluntária = risco jurídico elevado.

❌ Consequências

5. Usuário não pode pagar o Bradesco com saldo, nem sacar.

Isso força o usuário à quebra de contrato, mesmo após ter quitado outros.

É mau uso de serviço financeiro e pode configurar: 

Indução à inadimplência

Prática abusiva (CDC, Art. 39)

Risco de judicialização por danos morais e materiais

⚖️ Questões Jurídicas

6. O usuário tem direito a extorno via Pix?

Sim, se: 

O contrato foi rompido por impossibilidade de execução causada pela plataforma.

Há comprovação de que o valor foi pago, mas não utilizado como prometido.

6.1 Pode pedir juros pelo valor preso + assédio de cobradores?

Sim, com base em: 

CDC (Código de Defesa do Consumidor) Art. 42 e 39

Enriquecimento ilícito da instituição credora

Danos morais e lucros cessantes

Possibilidade de juros baseados na SELIC (9,25% ao ano à época)

📲 Versão 9.23 do App

7.1 Saque/transferência: OK

Corrigiram a falha. O fluxo está completo.

7.2 Código sênior?

Provavelmente. Resolveram falha grave de UX e compliance.

7.3 Código júnior?

Improvável. Mudança de arquitetura exige maior domínio técnico.

7.4 Redefinição de escopo/design?

Quase certo. A falha foi reconhecida e redesenharam o produto.

7.5 Correção de bug?

Mais provável que tenha sido uma “correção de arquitetura/fluxo funcional”, não um simples bug.

7.6 Alteração na API do Pix?

Possivelmente, ou integração com outra instituição de pagamento que permite saque/transferência (ex: via conta digital).

⚠️ Judicialização

Sua pergunta final é extremamente pertinente:

Isso abre brechas para judicialização?

Sim. Claramente. Você pode argumentar:

Venda casada de serviços financeiros não funcionais.

A empresa{s} cobradora, fragmenta a divida inicial em varios fdic's eh uma divisao de negocios do banco.

Retenção indevida de valores

Impossibilidade de cumprir contrato de forma unilateral

Fragmentação artificial de dívidas entre múltiplos cobradores

Indução à inadimplência e assédio moral

Não devolução de valores residuais ou pagos que não foram usados

📌 Possíveis Ações

Reclame no Banco Central (https://www.bcb.gov.br/acessoinformacao/reclamacoes)

Registre na Plataforma Consumidor.gov.br

Acionar o Procon

Ajuizar ação com: 

Comprovante do pagamento

Comprovação da impossibilidade de saque

Comprovação de quebra de contrato induzida

Prints e protocolos de atendimento

**Comment by GuerreiroPUSHINPAY** [comment]
Para todos ! Segue opção de PIX via API - pushinpay.com.br 
Melhores taxas do mercado
